pwnlib/rop/call.py

```diff
@@ -111,18 +111,21 @@ def __init__(self, value, address = 0):
             value = [value]
         self.values = []
         self.address = address
-        self.size = 0
+        for v in value:
+            if isinstance(v, (list, tuple)):
+                self.size += context.bytes
```

Comment:
A casual read of this might suggest `context.bytes * len(v)` but I haven't touched this code in forever so I have no idea

Reply:
It actually is `context.bytes`, since this is later replaced by a single pointer to the array. The array itself is then appended, just what `AppendedArgument` name suggests.